lua/keymappings.lua

```diff
@@ -154,6 +154,20 @@ function M.config()
     },
   }
 
+  lvim.lsp_keys = {
+    ["K"] = { "<cmd>lua vim.lsp.buf.hover()<CR>", "Show hover" },
+    ["gd"] = { "<cmd>lua vim.lsp.buf.definition()<CR>", "Goto Definition" },
+    ["gD"] = { "<cmd>lua vim.lsp.buf.declaration()<CR>", "Goto declaration" },
+    ["gr"] = { "<cmd>lua vim.lsp.buf.references()<CR>", "Goto references" },
+    ["gI"] = { "<cmd>lua vim.lsp.buf.implementation()<CR>", "Goto Implementation" },
+    ["gs"] = { "<cmd>lua vim.lsp.buf.signature_help()<CR>", "show signature help" },
+    ["gp"] = { "<cmd>lua require'lsp.peek'.Peek('definition')<CR>", "Peek definition" },
+    ["gl"] = {
+      "<cmd>lua require'lsp.handlers'.show_line_diagnostics()<CR>",
+      "Show line diagnostics",
+    },
+  }
+
```

Comment:
Can you please move this to the new config file for lsp-related settings? see: https://github.com/LunarVim/LunarVim/blob/0d4016ed438deec3b01cc0a531e66a6e7f6523e4/lua/lsp/config.lua#L1-L5

The new entry name would be `lvim.lsp.buffer_mappings`. 

While you're at it, make it explicit that these are for normal mode, so `lvim.lsp.buffer_mappings.normal_mode = { ... }` just so we don't have a regression when/if someone wants to add buffer keys for other modes.

Reply:
I have updated this in #1687